src/authentication.ts

```diff
@@ -1,133 +1,154 @@
-import { postWithXForm } from './httpUtils'
-import { generateCodeChallenge, generateRandomString } from './pkceUtils'
+import { postWithXForm } from "./httpUtils";
+import { generateCodeChallenge, generateRandomString } from "./pkceUtils";
 import type {
   TInternalConfig,
+  TPopupPosition,
   TPrimitiveRecord,
   TTokenRequest,
   TTokenRequestForRefresh,
   TTokenRequestWithCodeAndVerifier,
   TTokenResponse,
-} from './types'
+} from "./types";
 
-const codeVerifierStorageKey = 'PKCE_code_verifier'
-const stateStorageKey = 'ROCP_auth_state'
+const codeVerifierStorageKey = "PKCE_code_verifier";
+const stateStorageKey = "ROCP_auth_state";
```

Comment:
I can tell you right now that you should try to run the installed pre-commit hooks, as all these changes go against the style guide in the repo 😄 

We'll look into the changes when the changes aren't all over!